geno/.htaccess

```diff
@@ -4,7 +4,9 @@ RewriteEngine on
 
 RewriteRule ^CommentEnMer https://comment-enmer.github.io [R=302,L]
 
-RewriteRule ^$ https://gen-o.github.io/ [R=302,L]
+RewriteRule ^$ https://gen-o.github.io [R=302,L]
 
-RewriteRule (.*) https://gen-o.github.io/$1 [R=302,L,QSA]
+RewriteRule 2023-07-01 https://gen-o.github.io [R=302,L]
+
+RewriteRule website https://gen-o.github.io/website [R=302,L]
```

Comment:
Are you sure this is ok? I think you need a "$" to indicate the end. Like `website$`

Reply:
Yes, better, thanks a lot